daemon/mgr/network.go

```diff
@@ -65,13 +65,32 @@ type NetworkManager struct {
 }
 
 // NewNetworkManager creates a brand new network manager.
-func NewNetworkManager(cfg *config.Config, store *meta.Store) (*NetworkManager, error) {
+func NewNetworkManager(cfg *config.Config, store *meta.Store, ctrMgr ContainerMgr) (*NetworkManager, error) {
 	// Create a new controller instance
 	cfg.NetworkConfg.MetaPath = path.Dir(store.BaseDir)
 	cfg.NetworkConfg.ExecRoot = network.DefaultExecRoot
 
 	initNetworkLog(cfg)
 
+	// get active sandboxes
+	ctrs, err := ctrMgr.List(context.Background(),
+		func(c *Container) bool {
+			return (c.IsRunning() || c.IsPaused()) && !isContainer(c.HostConfig.NetworkMode)
+		}, &ContainerListOption{All: true})
+	if err != nil {
+		logrus.Errorf("failed to new network manager, can not get container list")
+		return nil, errors.Wrap(err, "failed to get container list")
+	}
+	cfg.NetworkConfg.ActiveSandboxes = make(map[string]interface{})
```

Comment:
do we need to store `ActiveSandboxes` to disk ?

Reply:
don't need to store to disk